build/commands/lib/config.js

```diff
@@ -1151,12 +1147,26 @@ Config.prototype.update = function (options) {
   }
 }
 
-Config.prototype.getTargetOS = function () {
-  if (this.targetOS) {
-    return this.targetOS
-  }
-  return this.hostOS
-}
+Object.defineProperty(Config.prototype, 'targetOS', {
+  get: function () {
+    if (this._targetOS) {
+      return this._targetOS
+    }
+    return this.hostOS
+  },
+  set: function (value) {
+    this._targetOS = value
+    if (this._targetOS) {
+      const supportedOS = ['android', 'ios', 'linux', 'mac', 'win']
+      assert(
```

Comment:
nit: we have about the same assert [here](https://github.com/brave/brave-core/blob/master/build/commands/lib/config.js#L985)
WDYT about removing it in favor of this one?

Reply:
it's removed in this PR already 😉